pkgs/servers/shairport-sync/default.nix

```diff
@@ -1,35 +1,73 @@
-{ lib, stdenv, fetchFromGitHub
-, autoreconfHook, pkg-config
-, openssl, avahi, alsa-lib, glib, libdaemon, popt, libconfig, libpulseaudio, soxr
-, enableDbus ? stdenv.isLinux
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, openssl_1_1
+, avahi
+, alsa-lib
+, libplist
+, glib
+, libdaemon
+, libsodium
+, libgcrypt
+, ffmpeg
+, libuuid
+, vim
+, popt
+, libconfig
+, libpulseaudio
+, libjack2
+, pipewire
+, soxr
```

Comment:
Please add

`, unixtools` here as well

Reply:
Sorry for the delay, been on vacation. Fixed (forgot to update when I switched from vim.xxd)